util/registries/github.ts

```diff
@@ -6,16 +6,16 @@ export interface GithubDatabaseEntry extends DatabaseEntry {
   type: "github";
   owner: string;
   repo: string;
+  default_version: string;
   path?: string;
-  default_version?: string;
```

Comment:
default_branch?

Reply:
Why does every entry have to specify it? Can't we assume master as default ?

Reply:
Can be. It's called `default_version` though because version is more universal than branch. (because `"type": "url"` also has a `default_version`?)

Reply:
Sorry - I thought this patch was adding the "default_version" feature - but it seems that already exists. This is making it required. Why is that desirable? It's more data in the database.json

Reply:
> Why does every entry have to specify it? Can't we assume master as default ?

That's what this branch PR changes. `default_version` is already supported on currently deployed site. The reason I wanted to remove the default being `master` is that people assume that we pull the default branch info from GitHub, so are confused when they don't use master as a default branch and everything breaks. Ref #1095 and #1130

Reply:
I don't particularly care about this - I'll approve it if you insist - it seems some documentation about the "default_version" in the readme would also solve the problem.

Reply:
I think most people don't read the README. Also there is a move towards not using `master` as the default git branch, so that kind of aligns with that too, because it makes us not opinionated to `master` being the main branch. I think explicit is better than implicit. But if you think its not worth it I can add docs to readme / pr template and close this.